Consul.Test/LockTest.cs

```diff
@@ -300,7 +300,10 @@ public async Task Lock_ContendFast()
                 }));
             }
 
-            await WithTimeout(Task.WhenAll(tasks));
+            // Set timeout to allow each task to wait up to the lock wait time
+            await WithTimeout(
+                Task.WhenAll(tasks),
+                TimeSpan.FromTicks(2 * contenderPool * Lock.DefaultLockWaitTime.Ticks));
```

Comment:
Now that we are using absolute timeouts everywhere else, why do we use a computed one here?
I think we should be consistent and either compute them everywhere, or nowhere. What do you think?